src/components/SharedComponents/Accordion.tsx

```diff
@@ -1,20 +1,24 @@
 import React from 'react';
-import { AccordionDetails } from '@material-ui/core';
+import { AccordionDetails, AccordionProps } from '@material-ui/core';
 import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
 import {
   StyledMuiAccordion,
   StyledAccordionSummary,
   MarkdownBody,
 } from './Accordion.style';
 
-const StyledAccordion = (props: {
+export interface StyledAccordionProps extends Omit<AccordionProps, 'children'> {
   summaryText: string;
   detailText: string;
-}) => {
-  const { summaryText, detailText } = props;
+}
 
+const StyledAccordion: React.FC<StyledAccordionProps> = ({
+  summaryText,
+  detailText,
+  ...otherProps
+}) => {
   return (
-    <StyledMuiAccordion>
+    <StyledMuiAccordion {...otherProps}>
```

Comment:
same as above q ^ :
does the updated syntax + addition of `{...otherProps}` serve a specific purpose here?